src/style.css

```diff
@@ -0,0 +1,141 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  padding: 12px;
+  font-family: 'calibri', sans-serif;
+  color: #777;
+}
+
+.to-do-container {
+  background-color: #eee;
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 1px 0;
+  margin: auto;
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add a short spacing at the top of your **to-do-container** as right now it is so close to the header. You can do it by just editing the `margin: auto;` value to something similar to this:
 `margin: 4rem auto;`